pkg/cli/cvd.go

```diff
@@ -412,3 +450,67 @@ func verifyCVDHostPackageTar(dir string) error {
 	}
 	return nil
 }
+
+type LocalArtifactsSrcs struct {
+	Images            map[string]string
+	Bootloader        string
+	CVDHostPackageTar string
+}
+
+func (s *LocalArtifactsSrcs) validate() error {
+	if s.Bootloader == "" {
+		return errors.New("missing bootloader source")
+	}
+	if s.CVDHostPackageTar == "" {
+		return errors.New("missing cvd host package tar source")
+	}
+
+	// TODO: Make it configurable.
+	var allowedImageKeys = []string{
+		"boot",
+		"init_boot",
+		"super",
+		"userdata",
+		"vbmeta",
+		"vbmeta_system",
+		"vbmeta_vendor_dlkm",
+		"vbmeta_system_dlkm",
+		"vendor_boot",
+	}
+	for k := range s.Images {
+		if !contains(allowedImageKeys, k) {
+			return fmt.Errorf("invalid image key: %q", k)
+		}
+	}
+	return nil
+}
+
+func (s *LocalArtifactsSrcs) names() []string {
+	result := []string{s.Bootloader, s.CVDHostPackageTar}
+	for _, v := range s.Images {
+		result = append(result, v)
+	}
+	return result
+}
+
+func loadLocalArtifacsSrcs(path string) (*LocalArtifactsSrcs, error) {
+	b, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
What about applying default configuration if user doesn't have toml file, but want to `cvdr create` with `local_artifacts`?

Reply:
hmm, it's not clear to me what a default config would look like here. what's your suggestion here?

Reply:
Conceptually, assuming someone set `--local_artifacts=default`, then `loadLocalArtifactsSrcs` parses the flag, recognizes `default`, and creates a default `LocalArtifactsSrcs` struct object not reading a file from given path.

On the other hand, I'm not certain whether setting default cfg is good idea or not, in case of aosp_cf someday requires another image(e.g. vendor.img) which default config doesn't know.